core/src/test/java/org/pentaho/platform/web/http/api/resources/services/SchedulerServiceTest.java

```diff
@@ -49,12 +51,8 @@
 
 import java.io.IOException;
 import java.io.Serializable;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Don't do star imports. This should be caught by the checkstyle. https://github.com/pentaho/pentaho-coding-standards?tab=readme-ov-file#checkstyle